js/ad_manager_vast.js

```diff
@@ -979,63 +1109,25 @@ OO.Ads.manager(function(_, $) {
      * @param {object} xml The xml returned from loading the ad
      */
     this._onVastResponse = function(adLoaded, xml) {
-      var vastAd = this.parser(xml);
-      if (!vastAd || !adLoaded) {
+      var vastAds = this.parser(xml);
+      if (!vastAds || !adLoaded || (_.isEmpty(vastAds.podded) && _.isEmpty(vastAds.standalone))) {
         this.errorType = "parseError";
         this.trigger(this.ERROR, this);
         failedAd();
-      }
-      else if (vastAd.type == "wrapper") {
-        this.currentDepth++;
-        if (this.currentDepth < OO.playerParams.maxVastWrapperDepth) {
-          var firstWrapperAd = vastAd.ads[0];
-          var _wrapperAds = this.wrapperAds;
-          OO.log("vast tag url is", firstWrapperAd.VASTAdTagURI, this.currentDepth);
-          if (firstWrapperAd) {
-            this.wrapperAds.error = this.wrapperAds.error.concat(firstWrapperAd.error);
-            this.wrapperAds.impression = this.wrapperAds.impression.concat(firstWrapperAd.impression);
-            this.wrapperAds.companion = this.wrapperAds.companion.concat(firstWrapperAd.companion);
-            this.wrapperAds.linear.ClickTracking = this.wrapperAds.linear.ClickTracking
-                .concat(firstWrapperAd.linear.ClickTracking);
-            _.each(firstWrapperAd.linear.tracking, function(value, key) {
-              _wrapperAds.linear.tracking[key] = _wrapperAds.linear.tracking[key] ?
-                                                 value.concat(_wrapperAds.linear.tracking[key]) :
-                                                 value;
-            });
-            _.each(firstWrapperAd.nonLinear.tracking, function(value, key) {
-              _wrapperAds.nonLinear.tracking[key] = _wrapperAds.nonLinear.tracking[key] ?
-                                                    value.concat(_wrapperAds.nonLinear.tracking[key]) :
-                                                    value;
-            });
-            if (!this.testMode) {
-              this._ajax(firstWrapperAd.VASTAdTagURI, this._onFinalError, 'xml');
-            } else {
-              this._handleLinearAd(adLoaded);
-              this._handleNonLinearAd(adLoaded);
-            }
-
-          }
-          else {
-            this.errorType = "wrapperParseError";
-            this.trigger(this.ERROR, this);
-            failedAd();
-          }
-        } else {
-          OO.log("Max wrapper depth reached.", this.currentDepth, OO.playerParams.maxVastWrapperDepth);
-          this.errorType = "tooManyWrapper";
-          this.trigger(this.ERROR, this);
-          failedAd();
+      } else {
+        var ad;
+        //TODO: Determine when we show standalone ads if podded ads are available
+        //Show podded ads if available
+        if(!_.isEmpty(vastAds.podded)) {
```

Comment:
I feel this conditional along with the conditional on LINE 1129 might read off better like:
```
if(_.isEmpty(vastAds.podded)) {
	ad = vastAds.standalone[0];
	if (ad) {
		handleAds([ad], adLoaded);
	}
}
else {
	handleAds(vastAds.podded, adLoaded);
}
```
that way the "branch"/flow for the standalone logic is grouped together.